packages/apps/crm/src/screens/lead/LeadListScreen.js

```diff
@@ -119,7 +119,11 @@ const LeadListScreen = ({navigation}) => {
               rightTitle={I18n.t('Crm_AssignedToMe')}
               onSwitch={() => setAssigned(!assigned)}
             />
-            <LeadSearchBar showDetailsPopup={false} oneFilter={true} />
+            <LeadSearchBar
+              showDetailsPopup={false}
+              oneFilter={true}
+              showTitle={false}
```

Comment:
No, LeadSearchBar should have default value to false for showTitle. The form view should set it to true manually